tests/foreman/cli/test_classparameters.py

```diff
@@ -194,12 +355,38 @@ def test_positive_update_parameter_type(self):
         3.  Submit the changes.
 
         @assert: Parameter Updated with a new type successfully.
-
-        @caseautomation: notautomated
         """
+        sc_param_id = self.sc_params_ids_list.pop()
+        for data in valid_sc_parameters():
+            with self.subTest(data):
+                SmartClassParameter.update({
+                    'parameter-type': data['sc_type'],
+                    'default-value': data['value'],
+                    'id': sc_param_id,
+                    'override': 1,
+                })
+                sc_param = SmartClassParameter.info({
+                    'puppet-class': 'ntp',
+                    'id': sc_param_id,
+                })
+                if data['sc_type'] == 'boolean':
+                    self.assertEqual(
+                        sc_param['default-value'],
+                        True if data['value'] == '1' else False
+                    )
+                elif data['sc_type'] == 'array':
+                    string_list = [
+                        str(element) for element in sc_param['default-value']]
+                    self.assertEqual(str(string_list), data['value'])
```

Comment:
Nope, we converted each element of array (from unicode to string) first and then here converted array as a whole to one string and compare with expected string